File: dev-docs/snippets/document-blob/get.swift
import JsBaoClient

func get(
  client: JsBaoClient,
  documentId: String,
  blobId: String
) async throws {
  // #region example
  let blobs = client.documents.blobs(documentId: documentId)
  // Returns a typed BlobInfo.
  let meta = try await blobs.get(blobId: blobId)
  let contentType = meta.contentType
  // #endregion example
  _ = (meta, contentType)
}

File: dev-docs/snippets/document-blob/get.ts
import type { JsBaoClient } from "js-bao-wss-client";

export async function get(
  client: JsBaoClient,
  documentId: string,
  blobId: string,
) {
  // #region example
  const blobs = client.document(documentId).blobs();
  const meta = await blobs.get(blobId);
  // #endregion example
  return meta;
}
